tests/workflow_templates/.gitignore

```diff
@@ -0,0 +1 @@
+*.yaml
```

Comment:
trying to wrap my head around this, is there a specific reason we needed to ignore yaml files here? 
workflow templates are generating yaml files?

Reply:
Yes, if you look at the integration tests, they generate the template and then submit it as a regular workflow